docker/runner.go

```diff
@@ -30,8 +30,8 @@ func (n *Runner) Run(f faas.Function) error {
 		return errors.New("please install 'docker'")
 	}
 
-	if f.Image == "" {
-		return errors.New("Function has no associated image.  Has it been built?")
+	if f.Image == "" || f.ImageWithDigest() == "" {
+		return errors.New("Function has no associated image and SHA356 hash. Has it been built?")
```

Comment:
SHA356?

Reply:
When 256 is not enough 😅  Good catch, thanks!

Reply:
Fixed